### PR TITLE
Geany 1.33

### PR DESCRIPTION
**Pull request by Mno-hime**
Run tested, plugins load and activate.

### CODE REVIEW

**Comment by alarcher** [comment]
You need to split in two PRs: first geany, then geany-plugins as the latter cannot build against geany 1.31. Thanks!

**Comment by Mno-hime** [comment]
Geany plugins 1.33: https://github.com/OpenIndiana/oi-userland/pull/4174

**Comment by alarcher** [comment]
Can you add the lv locale to the manifest?

File: components/editor/geany/manifests/sample-manifest.p5m [hunk 1]
```diff
@@ -263,6 +263,7 @@ file path=usr/share/locale/kk/LC_MESSAGES/geany.mo
 file path=usr/share/locale/ko/LC_MESSAGES/geany.mo
 file path=usr/share/locale/lb/LC_MESSAGES/geany.mo
 file path=usr/share/locale/lt/LC_MESSAGES/geany.mo
+file path=usr/share/locale/lv/LC_MESSAGES/geany.mo
```
Comment (alarcher) [inline components/editor/geany/manifests/sample-manifest.p5m:266]: This should be added to geany.p5m as well.

**Review by alarcher — APPROVED** [review]

**Pull request merged**